test/SourceKit/InterfaceGen/gen_objc_concurrency.swift

```diff
@@ -11,10 +11,15 @@
 // SWIFT-GEN-INTERFACE-NEXT:    internal func foo(_ operation: String, completionHandler handler: @escaping (Int) -> Void)
 // SWIFT-GEN-INTERFACE:         internal func foo(_ operation: String) async -> Int
 
+// SWIFT-GEN-INTERFACE:         @MainActor internal func mainActorMethod()
+
+
 // RUN: %sourcekitd-test -req=interface-gen -using-swift-args -header %S/../Inputs/concurrency/header_concurrency.h -- %s -Xfrontend -enable-objc-interop -import-objc-header %S/../Inputs/concurrency/header_concurrency.h -sdk %clang-importer-sdk | %FileCheck %s --check-prefix=OBJC-GEN-INTERFACE
 
 // But don't print @available if it was implicitly added to an imported Clang decl (rdar://76685011).
 // OBJC-GEN-INTERFACE-LABEL: class ClassWithHandlerMethod {
 // OBJC-GEN-INTERFACE-NOT:     @available
 // OBJC-GEN-INTERFACE:         func method(withHandler operation: String!, completionHandler handler: (@Sendable (Int) -> Void)!)
 // OBJC-GEN-INTERFACE:         func method(withHandler operation: String!) async -> Int
+
+// OBJC-GEN-INTERFACE:         @MainActor open func mainAcctorMethod()
```

Comment:
This... should fail? `Acctor` vs `Actor` and it looks like it is `mainActorMethod` above.